brainrender/atlas_specific/aba_gene_expression/api.py

```diff
@@ -118,8 +126,8 @@ def download_gene_data(self, gene):
         if self.genes is None:
             self.genes = self.get_all_genes()
         if str(gene) not in self.genes.id.values:
-            raise ValueError(
-                f"The gened {gene} is not in the list of available genes"
```

Comment:
Is it actually so common to get so many misuses of this class to justify the amount of custom errors you're throwing? What about leaving the upstream failing by itself?